### DISCUSSION TITLE
Testing Library

### DISCUSSION THREAD

**Discussion by joaopedromatias** [plouc/nivo#2229] [Q&A]
I'm not being able to render the chart inside the Testing Library. I'm using `jest-environment-jsdom`.

```js
// jest.config.js

module.exports = {
  globals: {
    'ts-jest': {
      isolatedModules: true,
    },
  },
  collectCoverage: true,
  testEnvironment: 'jest-environment-jsdom',
  testMatch: ['<rootDir>/app/**/*.test.[jt]s?(x)'],
}
```

I have checked [this issue](https://github.com/plouc/nivo/issues/2085), but even when mocking the `ResizeObserver` it does not renders. The div supposed to hold the chart is empty: 

![image](https://user-images.githubusercontent.com/90068133/215130095-44babad9-8870-4964-b368-60c09f65428b.png)

**Comment by rpereira-anchor** [comment]
https://github.com/plouc/nivo/issues/2232#issuecomment-1647355242
